cypress.config.ts

```diff
@@ -33,7 +33,7 @@ export default defineConfig({
       })
       return config
     },
-    baseUrl: 'http://localhost:3007',
+    baseUrl: 'http://localhost:3000',
```

Comment:
I don't think we are supposed to change this.
As there is a npm script - `npm run start-feature:dev` that uses `feature.dev` file which contains port 3007 like this:-
```
PORT=3007
REDIS_ENABLED=false
NODE_ENV=development
ENVIRONMENT_NAME=dev
```

Reply:
done but needs to be changed to 3000 when running tests

Reply:
Ok, I will raise a ticket for getting the tests to run on port 3007.